backend-elasticsearch/src/main/java/org/hibernate/search/v6poc/backend/elasticsearch/search/query/impl/SearchBackendContext.java

```diff
@@ -51,7 +50,11 @@ public SearchBackendContext(BackendImplementor<?> backend,
 
 	@Override
 	public String toString() {
-		return getClass().getSimpleName() + "[backend=" + backend + "]";
+		return getClass().getSimpleName() + "[" + failureContext + "]";
```

Comment:
Same here about linking `toString()` and failure contexts.

Reply:
Same answer, see above.